scripts/what_changed.py

```diff
@@ -0,0 +1,345 @@
+                "regex" : ["^subsys/net"],
+                "counter": 0,
+                "labels": ["area: Networking"]
+                },
+            {
+                "area": "Logging",
+                "regex" : ["^subsys/logging"],
+                "counter": 0,
+                "labels": ["area: Logging"]
+                },
+            {
+                "area": "Shell",
+                "regex" : ["^subsys/shell"],
+                "counter": 0,
+                "labels": ["area: Shell"]
+                },
+            {
+                "area": "Console",
+                "regex" : ["^subsys/console"],
+                "counter": 0,
+                "labels": ["area: Console"]
+                },
+            {
+                "area": "Testsuite",
+                "regex" : ["^subsys/testsuite"],
+                "counter": 0,
+                "labels": ["area: Testing Suite"]
+                },
+            {
+                "area": "Settings",
+                "regex" : ["^subsys/settings"],
+                "counter": 0,
+                "labels": ["area: Settings"]
+                },
+            {
+                "area": "File System",
+                "regex" : ["^subsys/fs"],
+                "counter": 0,
+                "labels": ["area: File System"]
+                },
+            {
+                "area": "Storage",
+                "regex" : ["^subsys/storage"],
+                "counter": 0,
+                "labels": ["area: Storage"]
+                },
+            {
+                "area": "Bluetooth",
+                "regex" : ["^subsys/bluetooth"],
+                "counter": 0,
+                "labels": ["area: Bluetooth"]
+                },
+            {
+                "area": "API",
+                "regex" : ["^include/"],
+                "counter": 0,
+                "labels": ["area: API"]
+                },
+            {
+                "area": "Samples",
+                "regex" : ["^samples/"],
+                "counter": 0,
+                "labels": ["area: Samples"]
+                },
+            {
+                "area": "Tests",
+                "regex" : ["^tests/"],
+                "counter": 0,
+                "labels": ["area: Tests"]
+                },
+            {
+                "area": "Kernel",
+                "regex" : ["^kernel/"],
+                "counter": 0,
+                "labels": ["area: Kernel"]
+                },
+            {
+                "area": "External",
+                "regex" : ["^ext/"],
+                "counter": 0,
+                "labels": ["EXT"]
+                },
+            {
+                "area": "Documentation",
+                "regex" : ["^doc/", "\.rst$", "\.txt$"],
+                "counter": 0,
+                "labels": ["area: Documentation"]
+                },
+            {
+                "area": "Build System",
+                "regex" : ["^cmake/", "^CMakeLists.txt", "\.cmake"],
```

Comment:
the last one should be removed. There are lots of `.cmake` files in the `boards/` folder but this should not set the "Build System" label

Reply:
For example the PR you linked should not have a "Build System" label IMHO: https://github.com/zephyrproject-rtos/zephyr/pull/16662/